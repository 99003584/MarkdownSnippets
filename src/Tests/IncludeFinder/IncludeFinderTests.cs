using MarkdownSnippets;
using Xunit;
using Xunit.Abstractions;

public class IncludeFinderTests :
    XunitApprovalBase
{
    [Fact]
    public void Simple()
    {
        var includes = new IncludeFinder().ReadIncludes("IncludeFinder");
        ObjectApprover.Verify(includes, Scrubber.Scrub);
    }

    public IncludeFinderTests(ITestOutputHelper output) :
        base(output)
    {
    }
}