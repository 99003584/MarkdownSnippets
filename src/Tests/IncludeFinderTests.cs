using System.Threading.Tasks;
using MarkdownSnippets;
using VerifyXunit;
using Xunit;

[UsesVerify]
public class IncludeFinderTests
{
    [Fact]
    public Task Simple()
    {
        IncludeFinder finder = new();
        var includes = finder.ReadIncludes("IncludeFinder");
        return Verifier.Verify(includes);
    }
}