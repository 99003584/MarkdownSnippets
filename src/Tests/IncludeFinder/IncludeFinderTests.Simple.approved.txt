[
  {
    Error: '',
    Key: 'file',
    Path: 'root\\IncludeFinder\file.include.md',
    Lines: [
      'Simple1'
    ]
  }
]